conductr_cli/conduct_services.py

```diff
@@ -24,40 +24,71 @@ def services(args):
     if log.is_verbose_enabled():
         log.verbose(validation.pretty_json(response.text))
 
-    data = sorted([
-                  (
-                      {
-                          'service': service,
-                          'bundle_id': bundle['bundleId'] if args.long_ids else bundle_utils.short_id(
-                              bundle['bundleId']),
-                          'bundle_name': bundle['attributes']['bundleName'],
-                          'status': 'Running' if execution['isStarted'] else 'Starting'
-                      }
-                  )
-                  for bundle in json.loads(response.text)
-                  for execution in bundle['bundleExecutions']
-                  for endpoint_name, endpoint in bundle['bundleConfig']['endpoints'].items() if 'services' in endpoint
-                  for service in endpoint['services']
-                  ], key=lambda line: line['service'])
+    def execution_status(bundle_executions):
+        for execution in bundle_executions:
+            if execution['isStarted']:
+                return 'Running'
+        return 'Starting'
+
+    def get_service_name_from_service_uri(service_uri):
+        paths = urlparse(service_uri).path.split('/')
+        if len(paths) > 1:
+            return paths[1]
+        else:
+            return ''
+
+    data_endpoint_services = [
+        (
+            {
+                'service_name': get_service_name_from_service_uri(service),
+                'service_uri': service,
+                'bundle_id': bundle['bundleId'] if args.long_ids else bundle_utils.short_id(
+                    bundle['bundleId']),
+                'bundle_name': bundle['attributes']['bundleName'],
+                'status': execution_status(bundle['bundleExecutions'])
+            }
+        )
+        for bundle in json.loads(response.text) if bundle['bundleExecutions']
+        for endpoint_name, endpoint in bundle['bundleConfig']['endpoints'].items() if 'services' in endpoint
+        for service in endpoint['services']
+    ]
+
+    data_endpoint_acls = [
```

Comment:
@huntc - can I call them `data_endpoint_with_services` and `data_endpoint_with_acls` instead?

Because `data_endpoint_with_services` is derived from the endpoints having services declaration, while `data_endpoint_with_acls` is derived from endpoints having acls declaration.